### DISCUSSION TITLE
Is there a way to pass an airflow connection for stores

### DISCUSSION THREAD

**Discussion by don1uppa** [fivetran/airflow-provider-great-expectations#132] [Q&A]
The great_expectations.yml has a credentials parameter for various stores.  Some of them support PostgreSQL, can that be supplied from airflow somehow?

**Comment by TJaniF** [comment]
Hi  @don1uppa
great question! No, at the moment you can only pass in credentials for your data sources as an Airflow connection (via the `conn_id` parameter), not for stores. I can create an issue for you to potentially add that in a feature version :)

**Comment by don1uppa** [comment]
Thanks Tamara, that would be great.

On Fri, Feb 23, 2024 at 10:18 AM Tamara Janina Fingerlin <
***@***.***> wrote:

> Hi @don1uppa <https://github.com/don1uppa>
> great question! No, at the moment you can only pass in credentials for
> your data sources as an Airflow connection (via the conn_id parameter),
> not for stores. I can create an issue for you to potentially add that in a
> feature version :)
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/astronomer/airflow-provider-great-expectations/discussions/132#discussioncomment-8569573>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AIA2BKVH6JWTHKNIQX2YSBDYVCXLFAVCNFSM6AAAAABDMIBGNKVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DKNRZGU3TG>
> .
> You are receiving this because you were mentioned.Message ID:
> <astronomer/airflow-provider-great-expectations/repo-discussions/132/comments/8569573
> @github.com>
>


-- 
Donald Iuppa
9934 Tioga Trail
Pinckney, Michigan 48169
-- and --
131 Eaglesfield Way
Fairport, NY 14450
***@***.***
Ph: (585) 210-8744

JOHN 11:19-27  Jesus said to her, “I am the resurrection and the life;
whoever believes in me, even if he dies, will live, and anyone who lives
and believes in me will never die.”
